app/components/route-details-header/route-details-header.tsx

```diff
@@ -89,27 +91,32 @@ export interface RouteDetailsHeaderProps {
    */
   screenName?: "routeDetails" | "activeRide"
   style?: ViewStyle
+  eventConfig?: AddCalendarEvent.CreateOptions
 }
 
 export const RouteDetailsHeader = observer(function RouteDetailsHeader(props: RouteDetailsHeaderProps) {
-  const { originId, destinationId, screenName, style } = props
+  const { originId, destinationId, screenName, style, eventConfig } = props
   const { favoriteRoutes } = useStores()
   const navigation = useNavigation()
 
+  const addToCalendar = (eventConfig) => {
+    AddCalendarEvent.presentEventCreatingDialog(eventConfig);
+  }
   const originName = stationsObject[originId][stationLocale]
   const destinationName = stationsObject[destinationId][stationLocale]
 
   const routeId = `${originId}${destinationId}`
 
   const isFavorite: boolean = useMemo(() => {
-    return favoriteRoutes.routes.find((favorite) => favorite.id === routeId)
+    return favoriteRoutes.routes.find((favorite) => favorite.id === routeId) !== undefined
```

Comment:
Why did you add that?

Reply:
find function is not returning a boolean value... usually its still working but here the lint is complaining